format/FlightSql.proto

```diff
@@ -933,6 +933,16 @@ message CommandGetDbSchemas {
  *  [optional] table_schema: bytes not null (schema of the table as described in Schema.fbs::Schema,
  *                                           it is serialized as an IPC message.)
  * >
+ * Fields on table_schema may contain the following metadata:
+ *  - ARROW:FLIGHT:SQL:CATALOG_NAME      - Table's catalog name
+ *  - ARROW:FLIGHT:SQL:DB_SCHEMA_NAME    - Database schema name
+ *  - ARROW:FLIGHT:SQL:TABLE_NAME        - Table name
+ *  - ARROW:FLIGHT:SQL:PRECISION         - Column precision/size
+ *  - ARROW:FLIGHT:SQL:SCALE             - Column scale/decimal digits
```

Comment:
Why are precision and scale useful? Can they annotation non-decimal columns?

Reply:
This is also requested by the JDBC and ODBC, to be extracted by them via ResultSetMetadata

Reply:
Should we annotate this as `(if applicable for the column type)` or something?

Reply:
It makes senses. Added